.jshintignore

```diff
@@ -0,0 +1,2 @@
+assets/javascripts/gridster/jquery.gridster.min.js
+assets/javascripts/jquery.knob.min.js
```

Comment:
yes, `vendor` would make sense. I am not always clear on what belongs in each. this seems like a helpful link: http://guides.rubyonrails.org/asset_pipeline.html#asset-organization